presto-tests/src/test/java/com/facebook/presto/server/TestServerInfoResource.java

```diff
@@ -43,22 +49,43 @@
 {
     private HttpClient client;
     private DistributedQueryRunner queryRunner;
+    private ThriftCodecManager thriftCodeManager;
+
+    private NodeState getNodeState(URI uri, boolean useThriftEncoding, Protocol thriftProtocol)
```

Comment:
Might be a  good idea to move the private method to bottom of the class to improve readability.